meta-resin-common/recipes-connectivity/openvpn/openvpn/prepare-openvpn

```diff
@@ -12,6 +12,11 @@ if [ ! -f $CONFIG_PATH ]; then
 	exit 1
 fi
 
+if [ "$MANAGED_IMAGE" == "false" ]; then
+	echo "prepare-openvpn: Avoid preparing openvpn on unmanaged images."
```

Comment:
There are other same references in code. Keeping it like this for consistency for now and we can fix the whole occurrences in another PR.